Makefile

```diff
@@ -16,6 +16,12 @@ ifeq ($(DUNE_PROFILE),)
 DUNE_PROFILE := dev
 endif
 
+ifeq ($(TMPDIR),)
+CODA_TMP_DIR := /tmp
+else
+CODA_TMP_DIR := $(TMPDIR)
+endif
```

Comment:
If you are fine using `TMPDIR` instead of `CODA_TMP_DIR` in the Makefile, you can condense this whole thing to just `TMPDIR ?= /tmp`. Then running either `TMPDIR=xxx make ...` or `make TMPDIR=xxx ...` will set that variable to something other than the default (`/tmp`).